tests/subsys/rtio/rtio_api/CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ project(rtio_api_test)
 
 target_sources(app PRIVATE src/test_rtio_spsc.c src/test_rtio_mpsc.c src/test_rtio_api.c)
 
+if(CONFIG_CPLUSPLUS)
```

Comment:
`CONFIG_CPLUSPLUS` is deprecated. Please use `CONFIG_CPP`.

Reply:
Thanks, I thought it was deprecated but when I looked I didn't see the select DEPRECATED so I assumed I didn't remember correctly. I just checked the docs page and sure enough it says to use CONFIG_CPP and maybe I was looking at an old version of the docs.